#
# This file is autogenerated by pip-compile with Python 3.8
# by the following command:
#
#    pip-compile --resolver=backtracking ./requirements.in
#
anyio==3.6.2
    # via httpcore
cachetools==5.2.0
    # via google-auth
certifi==2022.9.24
    # via
    #   httpcore
    #   httpx
    #   kubernetes
    #   requests
charset-normalizer==2.1.1
    # via requests
google-auth==2.15.0
    # via kubernetes
h11==0.14.0
    # via httpcore
httpcore==0.16.2
    # via httpx
httpx==0.23.1
    # via lightkube
idna==3.4
    # via
    #   anyio
    #   requests
    #   rfc3986
jinja2==3.1.2
    # via -r ./requirements.in
kubernetes==25.3.0
    # via -r ./requirements.in
lightkube==0.11.0
    # via -r ./requirements.in
lightkube-models==1.25.4.4
    # via
    #   -r ./requirements.in
    #   lightkube
markupsafe==2.1.1
    # via jinja2
oauthlib==3.2.2
    # via requests-oauthlib
oci-image==1.0.0
    # via -r ./requirements.in
ops==1.5.4
    # via -r ./requirements.in
pyasn1==0.4.8
    # via
    #   pyasn1-modules
    #   rsa
pyasn1-modules==0.2.8
    # via google-auth
python-dateutil==2.8.2
    # via kubernetes
pyyaml==6.0
    # via
    #   kubernetes
    #   lightkube
    #   ops
requests==2.28.1
    # via
    #   kubernetes
    #   requests-oauthlib
requests-oauthlib==1.3.1
    # via kubernetes
rfc3986[idna2008]==1.5.0
    # via httpx
rsa==4.9
    # via google-auth
six==1.16.0
    # via
    #   google-auth
    #   kubernetes
    #   python-dateutil
sniffio==1.3.0
    # via
    #   anyio
    #   httpcore
    #   httpx
tenacity==8.1.0
    # via -r ./requirements.in
urllib3==1.26.13
    # via
    #   kubernetes
    #   requests
websocket-client==1.4.2
    # via kubernetes

# The following packages are considered to be unsafe in a requirements file:
# setuptools
